crypto/jwx_es256k.go

```diff
@@ -0,0 +1,10 @@
+//go:build jwx_es256k
+// +build jwx_es256k
+
+package crypto
+
+import "github.com/lestrrat-go/jwx/v2/jwa"
+
+func init() {
+	AddSupportedAlgorithm(jwa.ES256K)
```

Comment:
I added a mention to #2564 so it can use this mechanism for adding the alg to the metadata.

Reply:
Ok, this code is in the master now.